joplin/base/migrations/0033_auto_20180927_0500.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.0.8 on 2018-09-27 05:00
+
+from django.db import migrations
+from pprint import pprint
+from wagtail.core.blocks.stream_block import StreamValue
+
+def better_steps(apps, schema_editor):
+    '''
+    We can't import the models directly as they may be a newer
+    version than this migration expects. We use the historical versions.
+    '''
```

Comment:
This comment is decently self explanatory, but to elaborate - this allows us to avoid importing models (which change) and instead use the model that is currently in the DB. In this specific scenario:

Original model - Orderable steps relation only
Final model - StreamFieldSteps only
Model at this point in time in the DB schema migration process - 🌟BOTH 🌟